index.ts

```diff
@@ -13,27 +14,27 @@ export function convert(bin: string, theme?: string) {
         const { files } = await SourceBin.get(bin);
         const [ file ] = files;
         const content = file.content
-            .replace(/^ {4}/gm, "  ")
-            .replace(/\r/g, '')
-            .split(/^/gms)
-            .map((val, i, array) => {
-                return `<line><num>${ i + 1 }</num><span>${
-                    val.replace(/</g, "&lt;")
-                        .replace(/</g, "&gt;")
-                }</span></line>`
-            })
-            .map((val, i, array) => {
-                const [ , maxLine ] = array[array.length - 1]
-                    .match(/<num>(\d+)<\/num>/) || [ undefined, '0' ]
-                const lineNumber = String(i + 1);
-                const line = lineNumber.padStart(maxLine.length, " ")
-                return val.replace(
-                    new RegExp(`<num>${ lineNumber }</num>`, "g"),
-                    `<num>${ line }</num>`
-                )
-            })
-            .join('')
-        const html = preset.replace(/{{ contents }}/, content)
+            // .replace(/^ {4}/gm, "  ")
```

Comment:
Hmm , you have commented it out , why not just remove it ? or make utils folder and put the stuff there ? if it is required it will be used 🤷 .

Reply:
The utils file can be something like this
```ts
export function parseData(str: string) {
	// code
}
```

Reply:
utils doesn't seem that necessary yet, there is not much of code that will go there, and i'm not really used to work with more than a single file unless i really need to.

Reply:
Don't worry about it, I can take care of, just yeet the commented code